frontend/src/pages/Profile/UserSkills/UserSkills.tsx

```diff
@@ -42,62 +40,60 @@ const UserSkills: React.FC<Props> = ({
-      <SearchWrapper>
-        <SearchSkill
-          disableUnderline
-          placeholder="Search"
-          inputProps={{ maxLength: 100 }}
-          fullWidth
-          startAdornment={
-            <InputAdornment position="start">
-              <Search color="disabled" fontSize="medium" />
-            </InputAdornment>
-          }
-          onKeyDown={(event) => {
-            checkSpace(event);
-          }}
-          onChange={(event) => {
-            searchSkillInList(event.target.value);
-          }}
-          onPaste={(event) => {
-            event.preventDefault();
-            checkPastedValue(event.clipboardData.getData('Text'));
-          }}
-          value={searchSkill}
-        />
-        <Divider />
-      </SearchWrapper>
-      <SkillsList>
-        {technologies && technologies.length ? (
-          technologies.map((techGroup, id) => {
-            const isDividerVisible = id < technologies.length - 1;
-            return (
-              <div key={`${techGroup.group.name}_id`}>
-                <SkillsListItem>
-                  <SkillTitle>
-                    {techGroup.isPrimary && <Star alt="primary" src={starContained} />}
-                    <Title>{techGroup.group.name || 'No group'}</Title>
-                  </SkillTitle>
-                  <SkillsInfoList>
-                    {techGroup.achievedSkills.map((skillInfo) => (
-                      <React.Fragment key={`${skillInfo.skill.name}_id`}>
-                        <SkillInfoContainer skillItem={skillInfo} />
-                      </React.Fragment>
-                    ))}
-                  </SkillsInfoList>
-                </SkillsListItem>
-                {isDividerVisible && <SkillsDivider />}
-              </div>
-            );
-          })
-        ) : (
-          <NoSkills>
-            <NoContent message={NO_SKILLS} size={Size.medium} />
-          </NoSkills>
-        )}
-      </SkillsList>
-    </Suspense>
+    <SearchWrapper>
+      <SearchSkill
+        disableUnderline
+        placeholder="Search"
+        inputProps={{ maxLength: 100 }}
+        fullWidth
+        startAdornment={
+          <InputAdornment position="start">
+            <Search color="disabled" fontSize="medium" />
+          </InputAdornment>
+        }
+        onKeyDown={(event) => {
+          checkSpace(event);
+        }}
+        onChange={(event) => {
+          searchSkillInList(event.target.value);
+        }}
+        onPaste={(event) => {
+          event.preventDefault();
+          checkPastedValue(event.clipboardData.getData('Text'));
+        }}
+        value={searchSkill}
+      />
+      <Divider />
+    </SearchWrapper>
+    <SkillsList>
+      {technologies && technologies.length ? (
+        technologies.map((techGroup, id) => {
+          const isDividerVisible = id < technologies.length - 1;
+          return (
+            <div key={`${techGroup.group.name}_id`}>
```

Comment:
as far as I understood, the group name is unique, so we need to add an `_id` as plain text?

```suggestion
            <div key={techGroup.group.name}> 
```

Reply:
yep, but it not my code. i just removed Suspense here. Аlso this component should be refаctored.